ui/src/datasets.tsx

```diff
@@ -171,304 +173,275 @@ export function Datasets() {
   );
 
   return (
-    <GridBox sx={{ overflowY: "auto" }}>
-      <ActionBar title="Datasets" backURL={"/underlays/" + underlay.name} />
-      <GridLayout height="auto" sx={{ py: 1, px: 5 }}>
-        <GridLayout cols="1fr 2fr" spacing={2}>
-          <GridLayout rows="auto 320px">
-            <GridLayout cols={3} fillCol={1} rowAlign="middle">
-              <GridBox sx={{ py: 2 }}>
-                <Typography variant="body1em">Cohorts</Typography>
-              </GridBox>
-              <GridBox />
-              <Button
-                startIcon={<AddIcon />}
-                variant="contained"
-                onClick={showNewCohort}
+    <GridLayout rows>
+      <ActionBar
+        title="Datasets"
+        subtitle={`Data source: ${underlay.name}`}
+        backURL={"/underlays/" + underlay.name}
+      />
+      <GridBox sx={{ overflowY: "auto" }}>
+        <GridLayout height="auto" sx={{ py: 1, px: 5 }}>
+          <GridLayout cols="1fr 2fr" spacing={2}>
+            <GridLayout rows="auto 320px">
+              <GridLayout cols fillCol={2} spacing={1} rowAlign="middle">
+                <GridBox sx={{ py: 2 }}>
+                  <Typography variant="body1em">Cohorts</Typography>
+                </GridBox>
+                <Tooltip title="Cohorts are groups of people with common traits">
+                  <InfoIcon sx={{ display: "flex" }} />
+                </Tooltip>
+                <GridBox />
+                <Button
+                  startIcon={<AddIcon />}
+                  variant="contained"
+                  onClick={showNewCohort}
+                >
+                  New cohort
+                </Button>
+                {dialog}
+              </GridLayout>
+              <Paper
+                sx={{
+                  py: 2,
+                  overflowY: "auto",
+                  display: "block",
+                  width: "100%",
+                  height: "100%",
+                }}
               >
-                New cohort
-              </Button>
-              {dialog}
-            </GridLayout>
-            <Paper
-              sx={{
-                py: 2,
-                overflowY: "auto",
-                display: "block",
-                width: "100%",
-                height: "100%",
-              }}
-            >
-              <GridBox sx={{ px: 1, overflowY: "auto" }}>
-                {cohorts.length === 0 ? (
-                  <Empty
-                    maxWidth="80%"
-                    title="Cohorts are groups of people with common traits"
-                    subtitle={
-                      <>
-                        <Link
-                          variant="link"
-                          underline="hover"
-                          onClick={showNewCohort}
-                          sx={{ cursor: "pointer" }}
+                <GridBox sx={{ px: 1, overflowY: "auto" }}>
+                  {cohorts.length === 0 ? (
+                    <Empty
```

Comment:
Do you need one of these `Empty` blocks for the concept sets too? Or maybe you left it out because we expect to always have prepackaged ones?

Reply:
Yeah, Demographics will always exist so it's not necessary.